dialect/sql/schema/inspect_test.go

```diff
@@ -280,38 +287,41 @@ func TestInspector_Tables(t *testing.T) {
 				require.NoError(t, err)
 				tables, err := inspect.Tables(context.Background())
 				require.Equal(t, tt.wantErr, err != nil, err)
-				tablesMatch(t, tables, tt.tables(drv))
+				tablesMatch(t, drv, tables, tt.tables(drv))
 			})
 		}
 	}
 }
 
-func tablesMatch(t *testing.T, got, expected []*Table) {
+func tablesMatch(t *testing.T, drv string, got, expected []*Table) {
 	require.Equal(t, len(expected), len(got))
 	for i := range got {
-		columnsMatch(t, got[i].Columns, expected[i].Columns)
-		columnsMatch(t, got[i].PrimaryKey, expected[i].PrimaryKey)
-		foreignKeysMatch(t, got[i].ForeignKeys, expected[i].ForeignKeys)
+		columnsMatch(t, drv, got[i].Columns, expected[i].Columns)
+		columnsMatch(t, drv, got[i].PrimaryKey, expected[i].PrimaryKey)
+		foreignKeysMatch(t, drv, got[i].ForeignKeys, expected[i].ForeignKeys)
 	}
 }
 
-func columnsMatch(t *testing.T, got, expected []*Column) {
+func columnsMatch(t *testing.T, drv string, got, expected []*Column) {
 	require.Equal(t, len(expected), len(got))
 	for i := range got {
 		c1, c2 := got[i], expected[i]
-		require.Equal(t, c1.Name, c2.Name)
-		require.Equal(t, c1.Nullable, c2.Nullable)
```

Comment:
sorry, couldn't resist fixing the order of `actual, expected` here .. the error message being wrong is really confusing.